src/targets/sourcePathResolver.ts

```diff
@@ -47,7 +85,7 @@ export abstract class SourcePathResolverBase<T extends ISourcePathResolverOption
 
     localPath = fixDriveLetter(localPath);
     logger.verbose(LogTag.RuntimeSourceMap, `Mapped remoteToLocal: ${remotePath} -> ${localPath}`);
-    return path.resolve(localPath);
+    return resolve(localPath);
```

Comment:
This was changed a few days ago and was causing the `posix -> win` test case to fail on osx (and would have failed win -> posix on windows). Fixed it here.